### PR TITLE
Fix typo in TypeScript 2.2.md

### PR DESCRIPTION
**Pull request by eugeneilyin**

### CODE REVIEW

**Comment by eugeneilyin** [comment]
@microsoft-github-policy-service agree

File: packages/documentation/copy/en/release-notes/TypeScript 2.0.md [hunk 1]
```diff
@@ -741,7 +741,7 @@ function test(x: Bar) {
 ## Private and Protected Constructors
 
 A class constructor may be marked `private` or `protected`.
-A class with private constructor cannot be instantiated outside the class body, and cannot be extended.
+A class with a private constructor cannot be instantiated outside the class body, and cannot be extended.
 A class with protected constructor cannot be instantiated outside the class body, but can be extended.
```
Comment (jakebailey) [inline packages/documentation/copy/en/release-notes/TypeScript 2.0.md]:
```suggestion
A class with a protected constructor cannot be instantiated outside the class body, but can be extended.
```

File: packages/documentation/copy/en/release-notes/TypeScript 4.1.md [hunk 2]
```diff
@@ -583,7 +583,7 @@ TypeScript 4.1 ships with a quick fix to help fix this break.
 In JavaScript, object spreads (like `{ ...foo }`) don't operate over falsy values.
 So in code like `{ ...foo }`, `foo` will be skipped over if it's `null` or `undefined`.
 
-Many users take advantage of this to spread in properties "conditionally".
+Many users take advantage of this to spread properties "conditionally".
```
Comment (jakebailey) [inline packages/documentation/copy/en/release-notes/TypeScript 4.1.md:586]: This one isn't wrong; it's "spreading in" props to something else.

**Review by jakebailey — CHANGES_REQUESTED** [review]

**Review by eugeneilyin — COMMENTED** [review]
LGTM

**Review by jakebailey — APPROVED** [review]

**Review by jakebailey — APPROVED** [review]

**Pull request merged**